src/plugins/data_source_management/public/components/cluster_selector/create_cluster_selector.tsx

```diff
@@ -1,11 +0,0 @@
-/*
```

Comment:
do we need to change file name as well? the file name still using the cluster_selector?

Reply:
yeah, the folder has been removed